docs/docs/releases/0.2.0.md

```diff
@@ -5,8 +5,9 @@
 Version 0.2.0 is Mathesar's first **beta** release and features a brand new [access control system](#permissions), massive [performance improvements](#performance), and an important [security fix](#data-files).
 
 !!! caution ""
-    **We do not fully support upgrading from previous versions to `0.2.0`.**  For more details see [below](#upgrading).
-
+    **We do not support upgrading from previous versions to `0.2.0`.**
```

Comment:
I think this is a bad way to frame this. You want this section to give someone who's currently running 0.1.7, and who wants to run 0.2.0 a way to proceed. This framing is far too scary and negative. You should at least add some material here along the lines of "if you're currently running 0.1.7, and you'd like to move to 0.2.0, you need to reinstall Mathesar, and then reconnect."

There's nothing accurate about the word "recover", and it's scary for users. Their data won't be deleted, and it won't go anywhere. If they're connected via a different client, they won't even notice anything via that other client. "reconnect" or something would be far better.

Reply:
> Their data won't be deleted, and it won't go anywhere.

Depends on what we call 'data', their PostgreSQL databases won't go anywhere but they lose all of Mathesar data including connected database configuration.

I agree perhaps "Recover" might not be the best wording, but I'm strongly against using "Upgrade". I'm onboard with workshopping better terms.

Reply:
I've used the term "Migrating to 0.2.0 from previous versions". I think this is a reasonable middleground.